### PR TITLE
Fix/quick order popup

### PR DESCRIPTION
**Pull request by NAVODYA123**
#### What does this PR do?

-  Remove go-to-checkout button from quick order add-to-cart popup

#### How to test it? 

1. Visit the given [workspace](https://cartdemo--sbdsefuat.myvtex.com/quickorder?__bindingAddress=catalog.sef.ao.sbd-nonprod.com/en-US/catalog-global)
2. Sign in to the account 
3. Select a sold to select account
4. Navigate to quick order page and scroll to the bottom of the page where the product search is placed
5. Search for product `av25 - 73483-00001` or any other product and select it
6. Click on the Add to Cart button
7. A pop up will be displayed at the bottom of the page(the clickable button in the popup is removed hence, it is not visisble)

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "vendor": "sbdsefprod",
+  "vendor": "sbdsefuat",
```
Comment (Jayendra88) [inline manifest.json]: Change vendor name to `sbdsefprod`

File: react/AutocompleteBlock.tsx [hunk 2]
```diff
@@ -126,12 +126,12 @@ const AutocompleteBlock: FunctionComponent<any & WrappedComponentProps> = ({
 
       message = resolveToastMessage(success, isNewItem)
 
-      action = success
-        ? {
-          label: translateMessage(messages.seeCart),
-          href: '/checkout/#/cart',
-        }
-        : undefined
+      // action = success
```
Comment (Jayendra88) [inline react/AutocompleteBlock.tsx]: Remove commented line

**Review by Jayendra88 — CHANGES_REQUESTED** [review]

File: manifest.json (hunk 1 above)
Comment (NAVODYA123) [reply manifest.json]: done

File: react/AutocompleteBlock.tsx (hunk 2 above)
Comment (NAVODYA123) [reply react/AutocompleteBlock.tsx]: done

**Review by Jayendra88 — APPROVED** [review]

**Pull request merged**